compiler/ballerina-lang/src/main/java/module-info.java

```diff
@@ -74,5 +74,6 @@
     exports io.ballerina.projects.plugins;
     exports io.ballerina.projects.internal.model; // TODO Remove this exports
     exports io.ballerina.projects.internal.environment; // TODO Remove these exports
+    exports io.ballerina.compiler.api.impl.symbols;
```

Comment:
It is not the intended usage to utilize the implementation classes. Instead we use the exposed public interfaces. I see that the reason to use the impl is to use `BallerinaUnionTypeSymbol` in `BallerinaSymbolServiceImpl.java`. Here we can use the `UnionTypeSymbol` which is the public interface. Hence shall we remove this export?